src/mscorlib/src/Interop/Windows/kernel32/Interop.Globalization.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
They do not have to cleaned up if they go in the right direction. For example, some of them are adding a new line at the end of the file that gets rid of the github red mark. No problem with keeping those.